qa/suites/krbd/unmap/conf.yaml

```diff
@@ -3,3 +3,5 @@ overrides:
     conf:
       client:
         rbd default features: 1 # pre-single-major is v3.13, so layering only
+    log-whitelist:
+      - slow request
```

Comment:
krbd/unmap doesn't do any I/O, it just maps and unmaps images.  Let's drop this hunk.

Reply:
done